### PR TITLE
WikiFactory: improve profiling of DB queries

### PR DESCRIPTION
**Pull request by macbre**
Add "submethod" to `$fname` argument when logging SQL queries

@michalroszka

### CODE REVIEW

File: extensions/wikia/WikiFactory/Loader/WikiFactoryLoader.php [hunk 1]
```diff
@@ -290,7 +290,7 @@ public function execute() {
 							"city_dbname"
 						),
 						($this->mCityID) ? array( "city_list.city_id" => $this->mCityID ) : array( "city_list.city_dbname" => $this->mCityDB ),
-						__METHOD__
+						__METHOD__ . '::domaindb'
```
Comment (wladekb) [inline extensions/wikia/WikiFactory/Loader/WikiFactoryLoader.php]: I guess the "db" suffix doesn't hold any meaning here as well as in 2 places below. Can we get rid of it?

Comment (macbre) [inline extensions/wikia/WikiFactory/Loader/WikiFactoryLoader.php]: It simply matches the naming of profiler sections (see line 271)

Comment (wladekb) [inline extensions/wikia/WikiFactory/Loader/WikiFactoryLoader.php]: I'd rename them too :) but I'll let you make a final call.

Comment (macbre) [inline extensions/wikia/WikiFactory/Loader/WikiFactoryLoader.php]: I'd simply keep it as it is :)

**Pull request merged**